src/Behat/Transliterator/SyncTool.php

```diff
@@ -0,0 +1,215 @@
+<?php
+
+namespace Behat\Transliterator;
+
+use RollingCurl\Request;
+use RollingCurl\RollingCurl;
+use Yaoi\Command;
+use Yaoi\Command\Option;
+use Yaoi\Http\Client;
+use Yaoi\String\Lexer\Parsed;
+use Yaoi\String\Lexer\Parser;
+use Yaoi\String\Lexer\Renderer;
+use Yaoi\String\Lexer\Token;
+use Yaoi\String\StringValue;
+use Yaoi\String\Parser as StringParser;
+
+/**
+ * Tool for converting char tables for Behat/Transliterator from Perl to PHP
+ *
+ * @package Behat\Transliterator
+ */
+class SyncTool extends Command
+{
+    public $libVer = '1.27';
+
+    private $tokenizer;
+    private $renderer;
+    private $phpTable;
+    private $itemIndex;
+    private $nonQuestionBoxFound;
+    private $block;
+
+    public function __construct()
+    {
+        $escape = array(
+            '\\}' => '}',
+            '\\\\' => '\\',
+            '\\{' => '{',
+            '\\@' => '@',
+            '\\$' => '$',
+        );
+
+        $this->tokenizer = new Parser();
+        $this->tokenizer->addLineStopper('#');
+        $this->tokenizer->addQuote('qq{', '}', $escape);
+        $this->tokenizer->addQuote('q{', '}', $escape);
+        $this->tokenizer->addQuote('"', '"');
+        $this->tokenizer->addQuote("'", "'");
+        $this->tokenizer->addBracket('[', ']');
+        $this->tokenizer->addDelimiter(';');
+
+        $this->renderer = new Renderer();
+        $this->renderer
+            ->setBindKey('-~z', 'z~-')
+            ->strip('#')
+            ->keepBoundaries('[');
+    }
+
+    static function setUpDefinition(\Yaoi\Command\Definition $definition, $options)
+    {
+        $options->libVer = Option::create()->setIsUnnamed()
+            ->setDescription('Version of Perl library');
```

Comment:
IMO, we should not use an option here. The version should be in a constant in the code here, and changing the version of the Perl library should be done by editing the command. This way, we can know which version of the Perl library was used for the sync by looking at the code of the repo.